e2e/bare-client-auth/driver.py

```diff
@@ -0,0 +1,12 @@
+import flwr as fl
+from pathlib import Path
+
+
+# Start Flower server
+hist = fl.server.start_driver(
+    server_address="127.0.0.1:9091",
+    config=fl.server.ServerConfig(num_rounds=3),
+    root_certificates=Path("certificates/ca.crt").read_bytes(),
+)
+
+assert hist.losses_distributed[-1][1] == 0
```

Comment:
While this is technically still possible, we should replace it with a `ServerApp`-based approach

Reply:
It's probably best to do this in a new PR